go/mysql/auth_server_static_test.go

```diff
@@ -120,14 +120,14 @@ func TestHostMatcher(t *testing.T) {
 	}
 }
 
-func TestStaticConfigHUP(t *testing.T) {
+func TestStaticConfigHUPWithRotation(t *testing.T) {
 	tmpFile, err := ioutil.TempFile("", "mysql_auth_server_static_file.json")
 	if err != nil {
 		t.Fatalf("couldn't create temp file: %v", err)
 	}
 	defer os.Remove(tmpFile.Name())
 	*mysqlAuthServerStaticFile = tmpFile.Name()
-
+	*mysqlAuthServerStaticReloadInterval = time.Millisecond * 10
```

Comment:
The nit I mentioned was about putting the number before the unit `10 * time.Millisecond`.

Also, you have to restore the previous value of any global flags you change. Here's an example of how I do it: https://github.com/vitessio/vitess/blob/master/go/vt/vttablet/tabletmanager/vreplication/controller_test.go#L242.